Jieun/routes/userRouter.js

```diff
@@ -1,9 +1,9 @@
 const express = require("express");
 const userController = require("../controllers/userController");
-
 const router = express.Router();
 
 router.post("/signup", userController.signUp);
+router.get("/login", userController.login);
```

Comment:
```suggestion
router.post("/login", userController.login);
```
login 정보를 받을 때는 body를 통해 받으므로 post 메소드를 통해 처리되어야합니다!